skrub/_expressions/_subsampling.py

```diff
@@ -0,0 +1,139 @@
+import numpy as np
+
+from .. import _dataframe as sbd
+from . import _evaluation, _expressions
+
```

Comment:
Can you add comments around here telling us (the reader of this code) what the following few symbols (the constant, the function, the objects) are for in the big-picture of the code organization?

I'm thinking about the next people exploring this code base

Reply:
I added a comment for the constant and as we just discussed IRL I'll add high-level comments in the module containing the base class that apply to this class and other expression nodes